keras/callbacks.py

```diff
@@ -456,7 +456,9 @@ def on_epoch_end(self, epoch, logs=None):
 
 
 class EarlyStopping(Callback):
-    """Stop training when a monitored quantity has stopped improving.
+    """Stop training when a monitored quantity has stopped improving and
```

Comment:
Please standardize the docstring; the docstring summary (top line) should start with a single line ending in a dot. More information can be added after a line break.